ta/pkcs11/src/processing_symm.c

```diff
@@ -835,6 +857,45 @@ enum pkcs11_rc step_symm_operation(struct pkcs11_session *session,
 		}
 		break;
 
+	case PKCS11_CKM_AES_CBC_PAD:
+		if (step == PKCS11_FUNC_STEP_FINAL ||
+		    step == PKCS11_FUNC_STEP_ONESHOT)
+			break;
+
+		if (!in_buf) {
+			EMSG("No input data");
+			return PKCS11_CKR_ARGUMENTS_BAD;
+		}
+
+		switch (function) {
+		case PKCS11_FUNCTION_ENCRYPT:
+			assert(proc->extra_ctx);
+
+			res = TEE_CipherUpdate(proc->tee_op_handle,
+					       in_buf, in_size,
+					       out_buf, &out_size);
+
+			output_data = true;
+			rc = tee2pkcs_error(res);
+
+			/* Update text size still in the OS buffer. */
+			if (rc == PKCS11_CKR_OK)
+				last_block->last_block_size = (in_size + last_block->last_block_size) & 0xF;
```

Comment:
Break line to fit in 80 char/line.
It does not produce a nice layout but we must conform to optee_os coding style.
Maybe 
```c
				last_block->last_block_size =
					(in_size +
					 last_block->last_block_size) & 0xF;
```